src/extension.ts

```diff
@@ -64,11 +66,15 @@ export function activate(context: vscode.ExtensionContext) {
 
   CircleViewerProvider.register(context);
 
+  MetadataViewerProvider.register(context);
```

Comment:
Could you remove this MetadataViewer-related lines?

Reply:
ok, I`ll apply that part.